docs/release-notes/8.3.asciidoc

```diff
@@ -1,10 +1,27 @@
 [[release-notes-header-8.3.0]]
 == 8.3
 
+[discrete]
+[[release-notes-8.3.3]]
+=== 8.3.3
+
+[discrete]
+[[bug-fixes-8.3.3]]
+==== Bug fixes and enhancements
+* Fixes a bug that prevented the *Create field* button from appearing in the Fields browser when you accessed it from a Timeline created using the Alerts page's *Open in timeline* button ({pull}135842[#135842]).
+* Removes the unsupported `matches` operator from the *Add Rule Exception* flyout ({pull}136340[#136340]).
+* Prevents Rule Execution Log events from being wrongly ordered when max events are hit and events are filtered by status ({pull}131675[#131675]).
+* Isolates the query used for the *First seen* and *Last seen* fields on the Users, Hosts, and Network details pages to improve page load times and UI performance ({pull}133539[#133539]).
+
 [discrete]
 [[release-notes-8.3.2]]
 === 8.3.2
 
+[discrete]
+[[known-issue-8.3.2]]
+==== Known issue
+* The `matches` operator in the *Add Rule Exception* flyout does not work because wildcard matches are not supported for rule exceptions. Using the `matches` operator will cause rule exceptions and their associated rules to fail. You can restore failed rules by deleting the unsupported exception and refreshing the rule ({issue}136340[#136340]).
```

Comment:
```suggestion
* The `matches` operator in the *Add Rule Exception* flyout does not work because wildcard matches are not supported for rule exceptions. Using the `matches` operator will cause rule exceptions and their associated rules to fail. You can restore failed rules by deleting unsupported exceptions and refreshing the rules ({issue}136340[#136340]).
```